x/twap/types/params.go

```diff
@@ -0,0 +1,46 @@
+package types
+
+import (
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	epochtypes "github.com/osmosis-labs/osmosis/v10/x/epochs/types"
+)
+
+// Parameter store keys.
+var (
+	KeyPruneEpochIdentifier = []byte("PruneEpochIdentifier")
+)
+
+// ParamTable for twap module.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+func NewParams(pruneEpochIdentifier string) Params {
+	return Params{
+		PruneEpochIdentifier: pruneEpochIdentifier,
+	}
+}
+
+// default twap module parameters.
+func DefaultParams() Params {
+	return Params{
+		PruneEpochIdentifier: "day",
```

Comment:
Suggest having "day" as a global constant `defaultPruneEpochIdentifier` so that we can later use it in tests